content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -302,8 +302,8 @@ To update Pods to a new spec in a controlled way, use a
 ### Isolating Pods from a ReplicaSet
 
 You can remove Pods from a ReplicaSet by changing their labels. This technique may be used to remove Pods
-from service for debugging, data recovery, etc. Pods that are removed in this way will be replaced automatically (
-assuming that the number of replicas is not also changed).
+from service for debugging, data recovery, etc. Pods that are removed in this way will be replaced automatically
+(assuming that the number of replicas is not also changed).
```

Comment:
This was done so this does not happen : 
![Uploading Screenshot 2020-10-10 at 01.41.56.png…]()

Reply:
Not sure if this change is needed. How does the text render in the preview page?